src/search/landmarks/landmark_cost_partitioning_heuristic.cc

```diff
@@ -144,14 +144,15 @@ class LandmarkCostPartitioningHeuristicFeature
             "which point the above inequality might not hold anymore.");
         document_note(
             "Optimal Cost Partitioning",
-            "To use ``cost_partitioning=optimal``, you must build the planner with LP "
```

Comment:
What is the reason to change from double backquotes to single? This gets translated to txt2tags, as far as I could see double backquotes means monospace but I could not find the single backquote.

Reply:
I wasn't aware that this is for actual formatting reasons. I just saw that we often use single back ticks in code comments when referring to variables in the code, so I ajusted here. I'll revert the change to not break the formatting in the documentation.